.env.development.local

```
@@ -1,3 +1,4 @@
 FAST_REFRESH=true
 SKIP_PREFLIGHT_CHECK=true
+FILESYSYEM_CACHE=true
```

Just noticed this typo

```suggestion
FILESYSTEM_CACHE=true
```

Should I do a PR to fix this?

---

It was added to check file system cache in webpack 5. But currently that property is unused. So it wont impact this version.